client/state/initial-state.js

```diff
@@ -224,7 +216,7 @@ function getInitialServerState( initialReducer ) {
 		return null;
 	}
 
-	const serverState = deserialize( window.initialReduxState, initialReducer );
+	const serverState = deserialize( initialReducer, omit( window.initialReduxState, '_timestamp' ) );
```

Comment:
Can we use something like

```js
const { _timestamp, ...reduxState } = window.initialReduxState;
const serverState = deserialize( initialReducer, reduxState );
```

to avoid introducing the usage of `_.omit`?

Reply:
OK, fixed by introducing a `deserializeStored` helper that omits by destructuring.